src/main/java/net/quedex/api/user/UserMessageSender.java

```diff
@@ -85,16 +85,56 @@ void sendOrderSpec(OrderSpec orderSpec) {
     }
 
     void sendBatch(List<? extends OrderSpec> batch) {
+        sendMessageQueued(() -> createBatchNode(batch));
+    }
+
+    void sendTimeTriggeredBatch(final long batchId,
```

Comment:
Inconsistent use of `final` - not used in previous methods and in the whole file/project.

Reply:
Removed finals.